libgadget/utils/openmpsort.c

```diff
@@ -1,6 +1,4 @@
-#ifdef _OPENMP
```

Comment:
So we will not compile without -fopenmp. Is this already the case?

Reply:
It is already the case, enforced by #error in allvars.h

Reply:
The _OPENMP guards were there for the clang static analyzer, but that has long since grown OMP support.